uaclient/files.py

```diff
@@ -350,6 +352,147 @@ def contract_id(self):
         return None
 
 
+class NoticesLabelID(NoticeFileDetails, Enum):
+    REBOOT_REQUIRED = NoticeFileDetails(label="reboot_required", order_id="10")
+    ENABLE_REBOOT_REQUIRED = NoticeFileDetails(
+        label="enable_reboot_required", order_id="11"
+    )
+    REBOOT_SCRIPT_FAILED = NoticeFileDetails(
+        label="reboot_script_failed", order_id="12"
+    )
+    FIPS_REBOOT_REQUIRED = NoticeFileDetails(
+        label="fips_reboot_required", order_id="20"
+    )
+    FIPS_SYSTEM_REBOOT_REQUIRED = NoticeFileDetails(
+        label="fips_system_reboot_required", order_id="21"
+    )
+    FIPS_INSTALL_OUT_OF_DATE = NoticeFileDetails(
+        label="fips_install_out_of_date", order_id="22"
+    )
+    FIPS_DISABLE_REBOOT_REQUIRED = NoticeFileDetails(
+        label="fips_disable_reboot_required", order_id="23"
+    )
+    FIPS_PROC_FILE_ERROR = NoticeFileDetails(
+        label="fips_proc_file_error", order_id="24"
+    )
+    FIPS_MANUAL_DISABLE_URL = NoticeFileDetails(
+        label="fips_manual_disable_url", order_id="25"
+    )
+    WRONG_FIPS_METAPACKAGE_ON_CLOUD = NoticeFileDetails(
+        label="wrong_fips_metapackage_on_cloud", order_id="25"
+    )
+    LIVEPATCH_LTS_REBOOT_REQUIRED = NoticeFileDetails(
+        label="lp_lts_reboot_required", order_id="30"
+    )
+    CONTRACT_REFRESH_WARNING = NoticeFileDetails(
+        label="contract_refresh_warning", order_id="40"
+    )
+    DAEMON_AUTO_ATTACH_LOCK_HELD = NoticeFileDetails(
+        label="daemon_auto_attach_lock_held", order_id="50"
+    )
+    DAEMON_AUTO_ATTACH_FAILED = NoticeFileDetails(
+        label="daemon_auto_attach_failed", order_id="51"
+    )
+    OPERATION_IN_PROGRESS = NoticeFileDetails(
+        label="operation_in_progress", order_id="60"
+    )
+
+
+class Notices:
```

Comment:
I think instead of a class, this can just be a module. I think the use I'd like to see is something like:

```python
from uaclient.files import notices

notices.add(Notices.FIPS_SYSTEM_REBOOT_REQUIRED)
notices.remove(Notices.FIPS_SYSTEM_REBOOT_REQUIRED)
notices.list()  # returns list of all Notices tuples on disk currently
for notice in notices.list():
    print(notice.message)
```